src/currencyservice/src/server.cpp

```diff
@@ -176,10 +176,10 @@ class CurrencyService final : public hipstershop::CurrencyService::Service
     std::string span_name = "CurrencyService/Convert";
     auto span =
         get_tracer("currencyservice")->StartSpan(span_name,
```

Comment:
Could you also update this in here?
https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/currencyservice.md#create-new-spans

Reply:
ah yes, thx, done!